pkg/kubectl/cmd/create/create_job.go

```diff
@@ -89,15 +91,30 @@ func NewCmdCreateJob(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *
 	cmdutil.AddValidateFlags(cmd)
 	cmdutil.AddDryRunFlag(cmd)
 	cmd.Flags().String("from", "", "The name of the resource to create a Job from (only cronjob is supported).")
+	cmd.MarkFlagRequired("from")
 
 	return cmd
 }
 
+func (o *CreateJobOptions) Validate() (err error) {
+	if o.Name == "" {
```

Comment:
Here `o.Name` can't be empty, since `len(args) == 1` and `args[0]` will be non-empty as well. `NameFromCommandArgs` has already got a stricter checking.

Reply:
As for this check, `create cluster{rol}` also same redundant check, so I updated in https://github.com/kubernetes/kubernetes/pull/63638. Could you please take a look?